apps/pie-docs/src/assets/js/show-more.js

```diff
@@ -0,0 +1,58 @@
+let shown = false;
+let showMoreButtons;
+
+window.addEventListener('DOMContentLoaded', () => {
+    showMoreButtons = document.querySelectorAll('[data-js="show-more"]');
+    const componentStatusTables = document.querySelectorAll('[class="c-componentStatusTable "]');
+
+    if (!showMoreButtons.length || !componentStatusTables.length) return;
+
+    // ensures status tables initially only show 6 columns each
+    componentStatusTables.forEach((el) => el.querySelectorAll('tbody tr').forEach((tr, index) => {
+        const rows = index >= 6 ? tr.style.display = 'none' : '';
+
+        return rows;
+    }));
+
+    showMoreButtons.forEach((button) => {
+        button.addEventListener('click', (event) => {
+            tableReveal(event.target);
+        });
+    });
+});
+
+const tableReveal = (elm) => {
+    // elm.id refers to app-table or web-table to differentiate
+    const componentStatusTable = document.querySelector(`[data-js="${elm.id}"]`);
+    const trs = componentStatusTable.querySelectorAll('tbody tr');
+    // selects h2 heading of section e.g. apps or web
+    const componentStatusTableSection = document.querySelector(`[id="${elm.id.split('-')[0]}"]`);
+
+    const hide = () => {
+        trs.forEach((tr, index) => {
+            const rows = index >= 6 ? tr.style.display = 'none' : '';
+
+            return rows;
+        });
+
+        elm.innerHTML = 'Show More <svg xmlns="http://www.w3.org/2000/svg" role="presentation" focusable="false" fill="#125fca" viewBox="0 0 16 16" height="16" width="16" class="c-pieIcon c-pieIcon--chevronDown u-iconFilled" aria-hidden="true"><path d="M2.82 5.044 8 10.399 13.197 5l.963.875-5.364 5.565a1.164 1.164 0 0 1-1.636 0L1.875 5.945l.945-.901Z"></path></svg>';
```

Comment:
Again, just a small thing, but rather than have hardcoded SVGs, you could use the filter that is used to display the pieIcons svgs from the library:

```
const pieIcons = require('../filters/pieIconsSvg')();
const chevronDownIcon = pieIcons.find('chevronDown');
```

Reply:
Hey! I did intend on using this approach originally, however, I'm not sure it's possible due to how the show-more button is rendered - because I'm loading this js module through a script tag, I get the error `ReferenceError: require is not defined` because the browser does not support the require() function 😞 I'm not sure how best to tackle it, so went for the hardcoded values for now - happy to look into this separately though as the search.js module could also benefit from this being able to be imported 👍